.openshift-ci/pre_tests.py

```diff
@@ -56,3 +57,15 @@ def run(self):
             check=True,
             timeout=PreSystemTests.START_PREFETCH_TIMEOUT,
         )
+
+
+class CollectionPreTest:
```

Comment:
[optional] I'd give this more verbose (java-style) name
```suggestion
class CollectionModeOverridePreTest:
```